### ISSUE TITLE
"programs"  tab is empty and "devices and scenes" not in alphabetic order

### ISSUE THREAD

**Issue by kj4qyf** [UniversalDevicesInc/UD-Mobile-iOS#3]
In "My Programs" there is no information. See picture
![Screenshot 2021-02-04 at 4 04 52 PM](https://user-images.githubusercontent.com/30532776/106955172-dc62f380-6702-11eb-8f19-86028b238dbe.png)

In "Devices and Scenes", the devices are not listed in alphabetic order. In my Administrative Console I have organized my devices and scenes in numbered folders (10, 11, 12, 13, etc). In the App these folders are not ordered.
![Screenshot 2021-02-04 at 4 07 58 PM](https://user-images.githubusercontent.com/30532776/106955449-411e4e00-6703-11eb-81d4-0b4f3d1a9e89.png)

**Comment by JavierRefuerzo** [comment]
Hi please click on the center of the list item for programs,  clicking on the chevron will show the status for a given list item. The program screenshot shown is for the top level program folder.

**Comment by kj4qyf** [comment]
Yes that works. I had clicked on the 'arrow' on the right. But same numeric/alphabetic issue as in "devices and scenes".

**Comment by kj4qyf** [comment]
BTW my "name" kj4qyf comes from another use in github. This is asbril@hotmail.com

**Comment by JavierRefuerzo** [comment]
Thanks asbril, I will rename the issue to sort order.

**Comment by JavierRefuerzo** [comment]
Added sort order by type then name in 0.0.7
